numba_dpex/dpnp_ndarray/dpnp_empty.py

```diff
@@ -34,7 +34,7 @@ def typer(shape, dtype=None, usm_type=None, sycl_queue=None):
             usm_type = parse_usm_type(usm_type)
 
         if nb_dtype is not None and ndim is not None and usm_type is not None:
-            return dpnp_ndarray_Type(
+            return DpnpNdarray(
                 dtype=nb_dtype, ndim=ndim, layout="C", usm_type=usm_type
```

Comment:
Unrelated to this PR, but the implementation of `dpnp_empty` is broken and needs to be fixed for compute follows data to work. It is not capturing the sycl_queue in the type and hard coding layout to "C".